### PR TITLE
K0s control plane downscaling

### PR DESCRIPTION
**Pull request by makhov**
Downscaling k0s controlplane nodes and docs

### CODE REVIEW

File: docs/capi-controlplane-bootstrap.md [hunk 1]
```diff
@@ -63,3 +63,19 @@ spec:
...
+## Downscaling the control plane
+
+**WARNING: Downscaling is a dangerous operation and should only be done if you know what you are doing.**
+
+Kubernetes using etcd as its backing store. It's crucial to have a quorum of etcd nodes available at all times. Always run etcd as a cluster of **odd** members.
+    
+When downscaling the control plane, you need firstly to deregister the node from the etcd cluster. This can be done by running the following command on the node that will be removed:
+
+```bash
+k0s etcd leave
```
Comment (jnummelin) [inline docs/capi-controlplane-bootstrap.md:76]: Should the user also stop k0s first?

**Review by jnummelin — APPROVED** [review]
One question on the docs, otherwise lgtm.

File: docs/capi-controlplane-bootstrap.md (hunk 1 above)
Comment (makhov) [reply docs/capi-controlplane-bootstrap.md:76]: Stopping k0s will stop etcd as well and `k0s etcd leave` won't work

Comment (jnummelin) [reply docs/capi-controlplane-bootstrap.md:76]: Ah, of course 🤦

**Pull request merged**